libraries/scopes/src/simplify.js

```diff
@@ -0,0 +1,22 @@
+exports.simplifyScopeExpression = (scope) => {
+  let scopeSet = [];
+  scopeSet.push(scope);
+  removeDuplicateScopes(scope);
+  console.log(scopeSet[0]);
+  return scopeSet[0];
+};
+
+function removeDuplicateScopes(scope){
+  if(Object.keys(scope).includes('AnyOf') || Object.keys(scope).includes('AllOf')){
+    const values = (Object.values(scope)[0]);
+    if(ifDuplicateExists(values)){
+      let unique = [...new Set(values)];
+      const uniqueValues = unique.join(', ');
```

Comment:
I'm not sure why the join here?  They should remain distinct strings..

Reply:
Yes, that is correct. But here are my observations with and without `join()`, without `join()`, the splice function creates a nested array. I will upload the logs here. 

Without `join(()`
![Capture1](https://user-images.githubusercontent.com/30138596/68645377-dac97080-053d-11ea-89c1-de0be55b21a9.JPG)

With join()
![2](https://user-images.githubusercontent.com/30138596/68645397-f16fc780-053d-11ea-8ea1-84c348c25bb7.JPG)

Reply:
Initially I tried it without the `join()` which resulted in nested arrays. Presently, this is my output.

![Capture1](https://user-images.githubusercontent.com/30138596/68645907-90e18a00-053f-11ea-9176-a3dc06972183.JPG)

They are two different strings now. The problem now is nested arrays. Phew!